src/css/profile/mobile/common/slider.less

```diff
@@ -57,8 +73,8 @@
 
 	&.ui-slider-active {
 		.ui-slider-handler {
-			width: 22 * @px_base;
-			height: 22 * @px_base;
+			min-width: 22 * @px_base;
```

Comment:
scrubber is alwas 22 when active why change it to "min-" ?

Reply:
Its parent has 'flex' value of display. Width can be smaller than required.